homeassistant/components/deconz/light.py

```diff
@@ -39,7 +39,22 @@
 DECONZ_GROUP = "is_deconz_group"
 EFFECT_TO_DECONZ = {
     EFFECT_COLORLOOP: LightEffect.COLOR_LOOP,
-    "None": LightEffect.NONE,
+    "none": LightEffect.NONE,
```

Comment:
Why did you change this to `"none"`?

Reply:
The deCONZ REST API uses lowercase, so it reports `none`. That's also what pydeconz returns from `supported_effects`.